src/Rules/Rule.php

```diff
@@ -9,4 +9,9 @@ public static function allClasses(): AllClasses
     {
         return new AllClasses();
     }
+
+    public static function noClass(): NoClass
```

Comment:
what about naming it NoClasses, for consistency with AllClasses?

Reply:
Because it would not be correct English. It's the same in Italian "tutte le classi"/"nessuna classe".

Reply:
I am far from being an expert but I think also NoClasses is correct. Looking at [java's ArchUnit](https://www.archunit.org/userguide/html/000_Index.html) they decided to used it, eg. 

```java
ArchRule rule = ArchRuleDefinition.noClasses()
    .that().resideInAPackage("..service..")
    .should().accessClassesThat().resideInAPackage("..controller..");

rule.check(importedClasses);
```